python/foglamp/services/core/api/filters.py

```diff
@@ -451,6 +451,8 @@ def _delete_keys_from_dict(dict_del: Dict, lst_keys: List[str], deleted_values:
             if parent is not None:
                 if dict_del['type'] == 'JSON':
                     i_val = json.loads(dict_del[k]) if isinstance(dict_del[k], str) else json.loads(json.dumps(dict_del[k]))
+                elif dict_del['type'] == 'script':
+                    i_val = ""
```

Comment:
why its set to "", can it be empty dict?

Reply:
configuration value always expect in string, irrespective of the data type